components/amorphic/lib/buildStartUpParams.js

```diff
@@ -19,28 +18,27 @@ function buildStartUpParams(configStore) {
     amorphicOptions.appList = rootCfg.get('applications');
 
     if (!rootCfg.get('application')) {
-        console.error(chalk.red('FATAL you did not define an application in your config.json file.'));
+        throw new Error('FATAL you did not define an application in your config.json file.');
     }
 
     amorphicOptions.appStartList = rootCfg.get('application').split(';');
     amorphicOptions.mainApp = amorphicOptions.appStartList[0];
     amorphicOptions.port = rootCfg.get('port') || 3000; // default to 3000 if not listed
 
     if (!amorphicOptions.appList) {
-        console.error(chalk.red('FATAL you did not define an applications list in your config.json file.'));
+        throw new Error('FATAL you did not define an applications list in your config.json file.');
     }
     for (let i = 0; i < amorphicOptions.appStartList.length; i++) {
         if (!amorphicOptions.appList[amorphicOptions.appStartList[i]]) {
-            console.error(chalk.red('FATAL your application: %s is not in the applications list in your root config.'), amorphicOptions.appStartList[i]);
+            throw new Error('FATAL your application: %s is not in the applications list in your root config.');
```

Comment:
I think you are missing the variable here `amorphicOptions.appStartList[i]`

Reply:
updated!